recipe/meta.yaml

```diff
@@ -2,8 +2,13 @@ package:
   name: tensorflow
   version: "0.9.0"
 
+source:
+  fn: v0.9.0.tar.gz
```

Comment:
Maybe prefix with `tensorflow-*`. Kind of good to have a unique name for local source caches. Again not a blocker. Just a suggestion.